### PR TITLE
Fixed Deployment Error 

### PR DESCRIPTION
**Pull request by Ayushmaanagarwal1211**
@sanjay-kv  Sir i fixed that and it should work please check 

#326

### CODE REVIEW

**Comment by sanjay-kv** [comment]
image

image
Could you look into the code again.

**Comment by Ayushmaanagarwal1211** [comment]
> image
> 
> image Could you look into the code again.

yes sir i can fix now

**Comment by Ayushmaanagarwal1211** [comment]
image

File: pages/user/[...name].js [hunk 1]
```diff
@@ -310,7 +310,7 @@ export default function Stats() {
                                     // align={column.align}
                                     style={{ verticalAlign: "middle" }}
                                   >
-        <a target="_blank"   className="text-green-500 text-center" href={row[5]}>{row[6]}</a>                              </div> 
+        <a target="_blank"  rel="noreferrer" className="text-green-500 text-center" href={row[5]}>{row[6]}</a>                              </div> 
                              
                             </div>
                             )}
```
Comment (sanjay-kv) [inline pages/user/[...name].js:316]: Everything looks fine on codespace

**Review by sanjay-kv — APPROVED** [review]

**Comment by sanjay-kv** [comment]
merged it , let me know any build fails

**Pull request merged**